moabb/datasets/ssvep_mamem1.py

```diff
@@ -167,7 +170,7 @@ def _get_single_subject_data(self, subject):
             raw = RawArray(data, info, verbose=False)
             montage = make_standard_montage("GSN-HydroCel-256")
             raw.set_montage(montage)
-            sessions[session_name] = raw
+            sessions[session_name] = {'run_0': raw}
```

Comment:
Why do you save it so?

Reply:
The EEG are stored in memory with MNE format, in a nested dict for each subject, with runs stored inside sessions elements. The rationale is that sessions are separate recordings that requires to setup the EEG cap and calibrating the system, while runs (or blocks) are recordings that are separated by breaks, subjects keeping their cap.